components/bsx/Create/CreateToken.vue

```diff
@@ -54,7 +54,7 @@
           label="mint.submit"
           :disabled="disabled"
           :loading="isLoading"
-          @click="submit" />
+          @click="() => submit()" />
```

Comment:
i believe you can just call `submit()` here without arrow function
```suggestion
          @click="submit()" />
```